pkg/controller/dataimportcron-controller.go

```diff
@@ -1434,16 +1484,18 @@ func (r *DataImportCronReconciler) setJobCommon(cron *cdiv1.DataImportCron, obj
 }
 
 func (r *DataImportCronReconciler) newSourceDataVolume(cron *cdiv1.DataImportCron, dataVolumeName string) *cdiv1.DataVolume {
-	var digestedURL string
 	dv := cron.Spec.Template.DeepCopy()
-	if isURLSource(cron) {
-		digestedURL = untagDigestedDockerURL(*dv.Spec.Source.Registry.URL + "@" + cron.Annotations[AnnSourceDesiredDigest])
-	} else if isImageStreamSource(cron) {
-		// No way to import image stream by name when we want specific digest, so we use its docker reference
-		digestedURL = "docker://" + cron.Annotations[AnnImageStreamDockerRef]
-		dv.Spec.Source.Registry.ImageStream = nil
-	}
-	dv.Spec.Source.Registry.URL = &digestedURL
+	if isCronRegistrySource(cron) {
```

Comment:
not familiar enough with the code but, no need to do anything here with pvc source?

Reply:
we use it as is.